code/dff.py

```diff
@@ -1,63 +1,82 @@
 import argparse
 import json
+import logging
 import os
 from datetime import datetime as dt
 from pathlib import Path
 from typing import Union
 
 import aind_ophys_utils.dff as dff
 import h5py
-import numpy as np
-from aind_data_schema.core.processing import (DataProcess, PipelineProcess,
-                                              Processing, ProcessName)
+from aind_data_schema.core.processing import DataProcess, ProcessName
+from aind_log_utils.log import setup_logging
 from scipy.stats import skew
 
 
-def write_output_metadata(
+def write_data_process(
     metadata: dict,
-    process_json_dir: str,
-    process_name: str,
     input_fp: Union[str, Path],
     output_fp: Union[str, Path],
-    start_date_time: dt,
+    unique_id: str,
+    start_time: dt,
+    end_time: dt,
 ) -> None:
     """Writes output metadata to plane processing.json
 
     Parameters
     ----------
     metadata: dict
         parameters from suite2p motion correction
-    input_fp: str
-        path to data input
+    raw_movie: str
+        path to raw movies
     output_fp: str
-        path to data output
+        path to motion corrected movies
+    unique_id: str
+        unique identifier
+    start_time: dt
+        start time of processing
+    end_time: dt
+        end time of processing
     """
-    with open(Path(process_json_dir) / "processing.json", "r") as f:
-        proc_data = json.load(f)
-    processing = Processing(
-        processing_pipeline=PipelineProcess(
-            processor_full_name="Multplane Ophys Processing Pipeline",
-            pipeline_url=os.getenv("PIPELINE_URL", ""),
-            pipeline_version=os.getenv("PIPELINE_VERSION", ""),
-            data_processes=[
-                DataProcess(
-                    name=process_name,
-                    software_version=os.getenv("VERSION", ""),
-                    start_date_time=start_date_time,
-                    end_date_time=dt.now(),
-                    input_location=str(input_fp),
-                    output_location=str(output_fp),
-                    code_url=(os.getenv("DFF_EXTRACTION_URL")),
-                    parameters=metadata,
-                )
-            ],
-        )
+    data_proc = DataProcess(
+        name=ProcessName.DF_F_ESTIMATION,
+        software_version=os.getenv("VERSION", ""),
+        start_date_time=start_time.isoformat(),
+        end_date_time=end_time.isoformat(),
```

Comment:
is the time format consistent between capsules? You don't seem to use `.isoformat()` in some of the others? Maybe it doesn't matter cause aind-data-schema reformats it consistently?

Reply:
Oh, I overlooked it in the oasis caspule, you likely have the `.isoformat()` in every caspule